ui/page/send/send_confirm_modal.go

```diff
@@ -115,6 +115,8 @@ func (scm *sendConfirmModal) broadcastTransaction() {
 }
 
 func (scm *sendConfirmModal) Handle() {
+	scm.modal.SetDisabled(scm.isSending)
```

Comment:
Where is the value of `scm.isSending` modified? That's where `scm.modal.SetDisabled` should be called too.